src/tools/line.js

```diff
@@ -0,0 +1,166 @@
+(function(d3, fc) {
+    'use strict';
+
+    fc.tools.line = function() {
+
+        var xScale = d3.time.scale(),
+            yScale = d3.scale.linear(),
+            value = fc.utilities.fn.identity,
+            keyValue = fc.utilities.fn.index,
+            label = value,
+            // specifies whether the value is domain or screen coordinates
```

Comment:
I know this pattern came from crosshairs but I now think I would expect this to be handled by supplying identity scales rather than a flag. Otherwise I'm a bit scared that over time this flag will end up being propagated everywhere the scale is. Happy to see this spun off as a separate issue though.